src/styles/_reboot.scss

```diff
@@ -38,8 +38,14 @@
 html {
   font-family: sans-serif; // 2
   line-height: 1.15; // 3
+  -webkit-text-size-adjust: none; // 4
```

Comment:
vendor prefix shoukld be done by the post css which we have. please try without and ispoect the css

Reply:
right, I will take a deep look at postCSS support for `text-size-adjust`

Reply:
Okay, `text-size-adjust` support seems to be there https://github.com/postcss/autoprefixer/issues/264

Expect Firefox for Android, which isn't include in the default browser list
https://github.com/postcss/autoprefixer/issues/496